src/Compilers/CSharp/Test/Emit/Emit/EmitMetadataTests.cs

```diff
@@ -1293,7 +1293,8 @@ private static void VerifyAutoProperty(PropertySymbol property, bool isFromSourc
             {
                 if (property is SourcePropertySymbol sourceProperty)
                 {
-                    Assert.True(sourceProperty.IsAutoPropertyWithGetAccessor);
```

Comment:
>Assert.True(sourceProperty.IsAutoPropertyWithGetAccessor); [](http://example.com/codeflow?start=20&length=58)

Inlining implementation in test might be Ok, but I would wait until after that is the only usage of the ```IsAutoPropertyWithGetAccessor``` property.